routes/items.js

```diff
@@ -0,0 +1,72 @@
+/*eslint-env node*/
+const express = require('express');
+const router = express.Router();
+const alasql = require('alasql');
+require('../item_db.js');
+require('../config/auth.js');
+const middlewares = require('../config/middlewares.js');
+const bunyan = require('bunyan');
+
+router.param('itemId', middlewares.itemId);
+const log = bunyan.createLogger({name: 'items'});
+
+// ----- Get all Items -----
+router.get('/items/', middlewares.loggedIn, (req, res, next) => {
+	const items = alasql('SELECT * FROM items WHERE userId=?', req.user.id);
```

Comment:
so let's do it right, as the model will grow. Routes code should be separated from actual data manipulation. so here we should actually call a method `getUserItems(userid)` from `/src/model/item`